cloudmarker/clouds/gcpcloud.py

```diff
@@ -75,9 +78,13 @@ def _get_resource(self, service_name, version='v1'):
                                credentials=self._credentials,
                                cache_discovery=False)
 
-    def _get_firewalls(self):
+    def _get_firewalls(self, projectId):
```

Comment:
The parameter name should be `project_id`. We don't use camelCase for variable or parameter names. Please make this change at other places in the code too.